packages/core/src/renderer/logger/file-transport.injectable.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { transports } from "winston";
+import directoryForLogsInjectable from "../../common/app-paths/directory-for-logs.injectable";
+import { loggerTransportInjectionToken } from "../../common/logger/transports";
+import windowLocationInjectable from "../../common/k8s-api/window-location.injectable";
+import currentlyInClusterFrameInjectable from "../routes/currently-in-cluster-frame.injectable";
+import { getClusterIdFromHost } from "../utils";
+
+const rendererFileLoggerTransportInjectable = getInjectable({
+  id: "renderer-file-logger-transport",
+  instantiate: (di) => {
+    let frameId: string;
+
+    const currentlyInClusterFrame = di.inject(
+      currentlyInClusterFrameInjectable,
+    );
+
+    if (currentlyInClusterFrame) {
+      const { host } = di.inject(windowLocationInjectable);
+      const clusterId = getClusterIdFromHost(host);
+
+      frameId = clusterId ? `cluster-${clusterId}` : "cluster";
+    } else {
+      frameId = "main";
+    }
+
+    return new transports.File({
+      handleExceptions: false,
+      level: "info",
+      filename: `lens-renderer-${frameId}.log`,
+      dirname: di.inject(directoryForLogsInjectable),
+      maxsize: 1024 * 1024,
+      maxFiles: 0,
+      tailable: true,
```

Comment:
Not sure what the expectation is here, but I tested with a file that was just under 1M in size and observed that when the file reached 1M nothing more was written to the file or any other file, and nothing more was written to the developer tools console either??

https://user-images.githubusercontent.com/40840436/215165042-11ae55cf-3529-4fb0-91a4-9baa9be15dea.mov

This is on Ubuntu 22.04. I artificially padded the file with logs to get it near 1M in size.

Reply:
Thanks! Expectation was that it would keep writing logs and older lines would be lost but I was mistaken. I'll increase the maxFiles to allow it to properly rotate files.